.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "extends": "airbnb-base",
+  "rules": {
+    "camelcase": 0,
+    "comma-dangle": 0,
+    "dot-notation": 0,
+    "func-names": 0,
+    "no-console": 0,
+    "no-extra-semi": 0,
+    "no-global-assign": 0,
```

Comment:
Instead of disabling the rule we could also whitelist all our global constants. So that we don’t allow new ones.